x/slashing/keeper/infractions.go

```diff
@@ -35,7 +35,8 @@ func (k Keeper) HandleValidatorSignature(ctx sdk.Context, addr cryptotypes.Addre
 	// This counter just tracks the sum of the bit array
 	// That way we avoid needing to read/write the whole array each time
 	previous := k.GetValidatorMissedBlockBitArray(ctx, consAddr, index)
-	missed := !signed
+	missed := !signed && !k.sk.IsJailed(ctx, consAddr) // don't update missed blocks when validator's jailed
```

Comment:
What effect (if any) will this have on the operation of this code in the single chain (non IS) case?

Reply:
In non IS cases, jailed validators won't have their missed blocks counter updated anymore. This was a possible scenario due to the TM validator update delay.

Reply:
This needs to be much clearer, and the functionality probably needs to be different as well.

Reply:
I agree. IMO we shouldn't overload the missed flag. Also, we shouldn't change the behavior for non-CCV validators.

Reply:
> In non IS cases, jailed validators won't have their missed blocks counter updated anymore. This was a possible scenario due to the TM validator update delay.

This is true, but only for 1-2 blocks, not enough to accumulate another downtime window. And it is irrelevant anyway due to the check at line 80.

Reply:
If this tiny effect in non IS case isn't wanted, we can just falsify the `IsJailed` function in the staking module. It was added for CCV so it wouldn't have any negative impact.